alacart/alacart/Views/Site/Form.cshtml

```diff
@@ -0,0 +1,37 @@
+﻿@model Restaurant
+
+@{
+    ViewData["Title"] = "Form";
+}
+
+@{
+    var action = Model == null ? "Add" : "Edit";
+
+}
+
+@if (action == "Add")
+{
+
+    <h1>Add a new restaurant</h1>
+
+}
+
+else
+{
+    <h1>Edit existing restaurant</h1>
+}
+
+
+<form asp-action="@action" asp-controller="site" method="post">
+
+    <div class="form-group"></div>
+    <label asp-for="Name"></label>
+    <input class="form-control" asp-for="Name" />
```

Comment:
don't forget to add `<span asp-validation-for="*"></span>` per each input you have associated to a property that contains a attribute like `[Required]`